src/main/webapp/resources/js/components/project/ProjectSelect.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Select, SelectProps, Tag, Typography } from "antd";
+import { LabeledValue } from "antd/lib/select";
+
+export type Project = { identifier: number; name: string };
+
+export interface ProjectSelectProps extends SelectProps {
+  projects: Project[];
```

Comment:
You need to describe all you parameters

Reply:
Reached out about this comment. Instructed to disregard.